lib/x86simdsort.cpp

```diff
@@ -265,3 +265,47 @@ DISPATCH_KEYVALUE_SORT_FORTYPE(int32_t)
 DISPATCH_KEYVALUE_SORT_FORTYPE(float)
 
 } // namespace x86simdsort
+//
+
+extern "C" {
+XSS_EXPORT_SYMBOL
+void keyvalue_qsort_float_uint32(float *key, uint32_t *val, uint32_t size)
```

Comment:
Is there a reason why some of these functions take the size of the array as a uint32_t instead of a size_t? I'd prefer they all be the same if there's no reason for them not to be.